<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计数器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>320</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonCntDn">
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonCntRst">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEditCnt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButtonCntUp">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string extracomment="114514">+1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="plainTextEditIcon"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButtonIcon">
       <property name="text">
        <string>i</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonCntUp</tabstop>
  <tabstop>pushButtonCntDn</tabstop>
  <tabstop>lineEditCnt</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCntUp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnCntUp_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCntRst</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnCntRst_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>-10</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCntDn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnCntDn_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>-41</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonIcon</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnIcon_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnCntUp_Click()</slot>
  <slot>btnCntRst_Click()</slot>
  <slot>btnCntDn_Click()</slot>
  <slot>btnIcon_Click()</slot>
 </slots>
</ui>
